### PR TITLE
person detailed view

### PR DESCRIPTION
**Pull request by rerowep**
To test:

Nedded: rero-ils with mef records

See: http://localhost:5002/persons/1

### CODE REVIEW

File: rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+{% endif %}
+{% endmacro %}
+
+{% block javascript %}
+{{ super() }}
+{% assets "rero_ils_tab_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
+<script type="text/javascript">
+$(document).ready(function(){
+    $('a[data-toggle="tab"]').on('click', function(e) {
+        localStorage.setItem('activeTab', $(this).attr('href'));
```
Comment (jma) [inline rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html]: A small comment is welcome. Is it for storing the opened tab?

File: rero_ils/modules/ext.py [hunk 2]
```diff
@@ -39,6 +40,10 @@ def __init__(self, app=None):
             app.add_template_filter(format_date_filter, name='format_date')
             app.add_template_filter(to_pretty_json, name='tojson_pretty')
             app.add_template_filter(item_status_text, name='item_status_text')
+            app.add_template_filter(
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/ext.py]: Idem for item_status_text, shell we define them globally or for each module?

File: rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
+{%- extends config.RECORDS_UI_BASE_TEMPLATE %}
+{%- block page_body %}
+<div class="container search-page">
+  <div class="row">
+    <div class="col-xs-1">
+      <img alt="{{ _(Person) }}" src="/static/images/icon_person.png" class="img-responsive icon-document"></img>
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html]: ok

File: rero_ils/modules/persons/views.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+from flask import Blueprint
+
+blueprint = Blueprint(
+    'persons',
+    __name__,
+    url_prefix='/persons',
+    template_folder='templates',
+    static_folder='static',
+)
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/persons/views.py]: see comment above

File: rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html (hunk 1 above)
Comment (rerowep) [reply rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html]: ok

File: rero_ils/modules/ext.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jma) [reply rero_ils/modules/ext.py]: A filter should be defined globally it is general enough, else locally.

File: rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html [hunk 5]
```diff
@@ -0,0 +1,151 @@
+{%- extends config.RECORDS_UI_BASE_TEMPLATE %}
+{%- block page_body %}
+<div class="container search-page">
+  <div class="row">
+    <div class="col-xs-1">
+      <img alt="{{ _(Person) }}" src="{{ url_for('static', filename='images/icon_person.png') }}" class="img-responsive icon-document"></img>
```
Comment (Garfield-fr) [inline rero_ils/modules/persons/templates/rero_ils/detailed_view_persons.html]: Remove </img>

**Pull request merged**